docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+[[add-fleet-server-kubernetes]]
+= Deploy Fleet Server on Kubernetes
+
+[NOTE]
+====
+If your {stack} is orchestrated by {eck-ref}[ECK], we recommend to deploy the {fleet-server} through the operator. That simplifies the process, as the operator automatically handles most of the resources configuration and setup steps.
+
+Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run Fleet-managed {agent} on ECK] for more information.
+====
+
+[IMPORTANT]
+====
+This guide assumes familiarity with Kubernetes concepts and resources, such as `Deployments`, `Pods`, `Secrets`, or `Services`, as well as configuring applications in Kubernetes environments.
+====
+
+To use {fleet} for central management, a <<fleet-server,{fleet-server}>> must
+be running and accessible to your hosts.
+
+You can deploy {fleet-server} on Kubernetes and manage it yourself.
+In this deployment model, you are responsible for high-availability,
+fault-tolerance, and lifecycle management of the {fleet-server}.
+
+To deploy a {fleet-server} on Kubernetes and register it into Fleet you will need the following details:
+
+* The *Policy ID* of a Fleet policy configured with the {fleet-server} integration.
+* A *Service token*, used to authenticate Fleet Server with Elasticsearch.
+* For outgoing traffic:
+** The *{es} endpoint URL* where the {fleet-server} should connect to, configured also in the {es} output associated to the policy.
+** When a private or intermediate Certificate Authority (CA) is used to sign the {es} certificate, the *{es} CA file* or the *CA fingerprint*, configured also in the {es} output associated to the policy.
+* For incoming connections:
+** A *TLS/SSL certificate and key* for the {fleet-server} HTTPS endpoint, used to encrypt the traffic from the {agent}s. This certificate has to be valid for the *{fleet-server} Host URL* that {agent}s use when connecting to the {fleet-server}.
+* Extra TLS/SSL certificates and configuration parameters in case of requiring <<mutual-tls,mutual TLS>> (not covered on this document).
+
+This document walks you through the complete setup process, organized into the following sections:
+
+* <<add-fleet-server-kubernetes-compatibility,Compatibility requirements>>
+* <<add-fleet-server-kubernetes-cert-prereq>>
+* <<add-fleet-server-kubernetes-preparations>>
+* <<add-fleet-server-kubernetes-install>>
+
+[discrete]
+[[add-fleet-server-kubernetes-compatibility]]
+= Compatibility
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later.
+** For version compatibility: {es} >= {fleet-server} >= {agent} (except for
+bugfix releases)
```

Comment:
```suggestion
** For version compatibility, {es} must be at the same or a later version than {fleet-server}, and {fleet-server} needs to be at the same or a later version than {agent} (not including patch releases). 
```
Personally, I'd probably write this out rather than use the `>=` signs, but I think either one is just fine.

Reply:
Totally agreed. As mentioned in the previous comment, the compatibility section was copied from the other docs about fleet server installation, so we should probably fix that everywhere.

Reply:
I can make that change in the other docs.